backend/game/player/test_player.py

```diff
@@ -13,6 +13,21 @@ def player():
     return player
 
 
+def test_set_monster_name(player):
+    player.set_monster_name('Godzilla')
+    assert player.monster_name == 'Godzilla'
+
+
+def test_set_username(player):
+    player.set_username('Tester')
+    assert player.username == 'Tester'
+
+
+def test_set_password(player):
+    player.set_password('12345678')
```

Comment:
I don't think we should be storing plain text passwords for users. Are we planning to implement authentication? If so l, I think we should handle that outside of the player class. I don't think passwords should go in these database objects and can just be removed for the scope of this PR. If we used some external authentication provider (like sign in with google) we wouldn't even need passwords I would think

Reply:
password reference removed